index.js

```diff
@@ -49,17 +49,22 @@ function createStore (modules) {
     if (events[event]) {
       var changes = { }
       var changed, key
+      var newState = { }
+
+      for (key in state) newState[key] = state[key]
```

Comment:
If an event didn’t change the state, we will do this work without a reason.

It is not a problem, but I need some reason for it.

Reply:
If we move this logic inside and there are n listeners, we will have to do this n times.

Reply:
Good reason.